zebedee-cms/src/main/java/com/github/onsdigital/zebedee/session/store/exceptions/SessionsDecodeException.java

```diff
@@ -0,0 +1,10 @@
+package com.github.onsdigital.exceptions;
```

Comment:
The namespacing of this package looks incorrect. Same with the other exception packages you've created.